definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -6272,13 +6237,6 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "18/12/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "UpdateLanguage",
```

Comment:
Just need to keep this original object in "AuthorisationCaseEvent-prod.json".

Ideally, **if** Jeremy confirms every role should have "R" access. it would be to have the following changes:
inside prod file:
- original authorisations
inside nonprod file:
- caseworker-divorce-solicitor role with R access

Reply:
Update: 05bd41b5e745151c95270d63b46f4bcab1b6f007